pkg/embedded/templates/m2/settings.xml

```diff
@@ -0,0 +1,28 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+     <pluginGroups>
+         <pluginGroup>dev.galasa</pluginGroup>
+     </pluginGroups>
+     
+     <profiles>
+         <profile>
+             <id>galasa</id>
+             <activation>
+                 <activeByDefault>true</activeByDefault>
+             </activation>
+             <repositories>
+                 <repository>
+                     <id>galasa.repo</id>
+                     <url>https://development.galasa.dev/main/maven-repo/obr</url>
```

Comment:
Should point to the official released repo, not the development one ?

Reply:
This was what was created when you initialise the Galasa home folder from eclipse. But I will change this

Reply:
Yea. I don't think we generally want to point at the bleeding-edge stuff. That should be an opt-in rather than a default position.

Reply:
could that be flag to add such as 'galasactl local init --latest' or just having the comment in the settings.xml is enough?

Reply:
I think we should add it as a comment in the settings.xml.
Flags are pretty useless in a one-use command, nobody is going to run it again after the event. And the tool isn't clever enough to update the setting if the file already exists...